xml/snmp_sysdescr.xml

```diff
@@ -1564,6 +1564,20 @@
     <param pos="2" name="hw.product"/>
   </fingerprint>
 
+  <fingerprint pattern="^(ce\d+\S*)$">
```

Comment:
We may want to try test it out locally. The SNMP output would look like below and it seems to be one string with newlines. If so then we'll need to adjust the regex. 
```
SNMPv2-MIB::sysDescr.0 = STRING: Cisco Codec
SoftW: ce9.15.0.d97b604b745
MCU: Cisco Webex Room Kit
Date: 2020-10-26
S/N: ...
```
A different, simpler, example for Cisco Meraki, which is a one-liner:
`SNMPv2-MIB::sysDescr.0 = STRING: Meraki MX84 Cloud Managed Router`
and similarly complex for Cisco IOS:
```
SNMPv2-MIB::sysDescr.0 = STRING: Cisco IOS Software, C800 Software (C800-UNIVERSALK9-M), Version 15.6(3)M4, RELEASE SOFTWARE (fc1)
Technical Support: http://www.cisco.com/techsupport
Copyright (c) 1986-2018 by Cisco Systems, Inc.
Compiled Mon 05-Feb-18 11:06 by prod_rel_team
```

Reply:
Agree, we should make the regular expression more specific. The example also should be updated to contain the actual output of the SNMP sysDescr.